vcr/ambassador.go

```diff
@@ -63,9 +62,20 @@ func NewAmbassador(networkClient network.Transactions, writer Writer, verifier v
 }
 
 // Configure instructs the ambassador to start receiving DID Documents from the network.
-func (n ambassador) Configure() {
-	n.networkClient.Subscribe(network.TransactionPayloadAddedEvent, types.VcDocumentType, n.vcCallback)
-	n.networkClient.Subscribe(network.TransactionPayloadAddedEvent, types.RevocationLDDocumentType, n.jsonLDRevocationCallback)
+func (n ambassador) Configure() error {
+	err := n.networkClient.Subscribe("vcr_vcs", n.handleNetworkVCs,
+		n.networkClient.WithPersistency(),
+		network.WithSelectionFilter(func(event dag.Event) bool {
+			return event.Type == dag.PayloadEventType && event.Transaction.PayloadType() == types.VcDocumentType
```

Comment:
fine as it is. Even more awesome version would define (and combine) filters  for event types and payload types.